source/localizable/tutorial/routes-and-templates.md

```diff
@@ -218,21 +213,12 @@ Let's update our `index.hbs` with some HTML for our home page and our links to t
 
 In addition to providing button-style links in each route of our application, we would like to provide a common banner to display both the title of our application, as well as its main pages.
 
-When you create an Ember application with Ember CLI as we did, it generates a template called `application.hbs`.
-Anything you put in this template is shown for every page in the application.
-The default `application.hbs` file contains an `h2` tag with the text "Welcome to Ember", and an [`{{outlet}}`](http://emberjs.com/api/classes/Ember.Templates.helpers.html#method_outlet).
-The `{{outlet}}` defers to the router, which will render in its place the markup for the current route.
-
-```app/templates/application.hbs
-<h2 id="title">Welcome to Ember</h2>
+First, create a new file called `app/templates/application.hbs`.
```

Comment:
@toddjordan We encouraged folks to use the `ember g template application` approach in the quick start, can we do that here?